Formula/jnettop.rb

```diff
@@ -21,11 +21,19 @@ class Jnettop < Formula
   depends_on "pkg-config" => :build
   depends_on "glib"
 
+  uses_from_macos "libpcap"
+
   def install
     # Work around "-Werror,-Wimplicit-function-declaration" issues with
     # configure scripts on Xcode 12:
     ENV.append "CFLAGS", "-Wno-implicit-function-declaration"
 
+    # Fix undefined reference to `g_thread_init'
+    on_linux do
+      inreplace "Makefile.in", "$(jnettop_LDFLAGS) $(jnettop_OBJECTS)",
+                               "$(jnettop_OBJECTS) $(AM_LDFLAGS) $(LDFLAGS) $(jnettop_LDFLAGS)"
```

Comment:
Noting that this will compile with duplicated flags (the original `AM_LDFLAGS` and `LDFLAGS` are set in another variable), though that doesn't cause an issue.

Reply:
That should be fine.  This issue with LDFLAG ordering will go away when we move to Ubuntu 18.04 anyway.  Glad to see this worked!